de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/dvlm/concepts/registry/ActiveConceptConfigurationElement.java

```diff
@@ -143,6 +145,15 @@ private static Concept createActiveConcept(Concept concept, Repository repositor
 			
 			@Override
 			public EObject get(Object key) {
+				
+				// For EReferences to external EClasses ignore concept 
+				// activation and convert URIs instead

```

Comment:
maybe give a bit more explanation on why the uri is translated. I think it is a very important aspect.